client-web/src/exlibris-app/shell/redux-store.html

```diff
@@ -2,16 +2,28 @@
 
 <script>
     {
+        /*global Redux PolymerRedux ReduxMixin:true*/
```

Comment:
ReduxMixin is not global. It's inside of ExlibrisApplication.
BTW I'd suggest more short naming: ExlibrisApp.